code/modules/projectiles/guns/projectile/automatic/ak47.dm

```diff
@@ -98,8 +98,8 @@
 	w_class = ITEM_SIZE_NORMAL
 	force = WEAPON_FORCE_NORMAL
 	matter = list(MATERIAL_PLASTEEL = 15, MATERIAL_PLASTIC = 5)
-	price_tag = 650
-	recoil_buildup = 13
+	price_tag = 500
+	recoil_buildup = 16
```

Comment:
Could you mention these balance changes in the PR body ? Thanks.

Reply:
corrected